cmd/service_deploy.go

```diff
@@ -73,31 +76,40 @@ func init() {
 
 	serviceDeployCmd.Flags().BoolVar(&flagServiceDeployDockerComposeImageOnly, "image-only", false, "Only deploy the image when a docker-compose.yml file is specified.")
 
+	serviceDeployCmd.Flags().BoolVar(&flagServiceDeployWaitForService, "wait-for-service", false, "Wait for the service to reach a steady state after deploying the new task definition.")
```

Comment:
should we add a `-w` shorthand for this?

Reply:
I'm not opposed to it, I only left it out cause my node brain is so used to `-w` meaning `--watch`

Reply:
I think we should do it.